phpunit/fonts-api/wpPrintFonts-test.php

```diff
@@ -168,7 +183,80 @@ public function data_should_print_all_registered_fonts_for_iframed_editor() {
 						'Source Serif Pro-900-italic',
 					),
 					'output' => sprintf(
-						"<style id='wp-fonts-local' type='text/css'>\n%s%s%s\n</style>\n",
+						"<style id='wp-fonts-local'>\n%s%s%s\n</style>\n",
+						$font_faces['merriweather-200-900-normal'],
+						$font_faces['Source Serif Pro-300-normal'],
+						$font_faces['Source Serif Pro-900-italic']
+					),
+				),
+			),
+		);
+	}
+
+	/**
+	 * @dataProvider data_should_print_enqueued_fonts_for_non_site_editor
```

Comment:
```suggestion
	 * Enqueued fonts should be printed, but not other unused registered fonts.
	 *
	 * @dataProvider data_should_print_enqueued_fonts_for_non_site_editor
```

Reply:
Same here. 

How about this instead?

```php
test_should_print_only_enqueued_font_when_not_in_site_editor
```

This name follows the suggestion above and adds "only" to emphasize that only "enqueued" fonts are printed.

@ironprogrammer what do you think?